api/submission/routes.go

```diff
@@ -12,10 +12,11 @@ var routes []*core.Route = []*core.Route{
     core.NewAPIRoute(core.NewEndpoint(`submission/fetch/scores`), HandleFetchScores),
     core.NewAPIRoute(core.NewEndpoint(`submission/fetch/submission`), HandleFetchSubmission),
     core.NewAPIRoute(core.NewEndpoint(`submission/fetch/submissions`), HandleFetchSubmissions),
+    core.NewAPIRoute(core.NewEndpoint(`submission/fetch/history`), HandleFetchSubmissionHistory),
```

Comment:
I don't like this endpoint, too close to submission/history.

Let's think of an alternate name to replace "history". results? records? versions? attempts?
Or, I could maybe be persuaded for submission/fetch/submission/history.

(And of course, renaming here implies renaming files/functions.)

Reply:
Hmm...I like attempts and I like submission/fetch/submission/history.  I think they both communicate their intention.  Attempts make sense to me from a student standpoint.  My argument for the latter would be that both fetch/submission and fetch/submissions return grading results.  As such, I think that submission/fetch/submission/history communicates that we are fetching a history of grading results.   What do you think?

Reply:
Make this attempts

Reply:
(As discussed live.)
Let's go with attempts: `submission/fetch/attempts`.